### ISSUE TITLE
Updated json correction from MUO-POG

### ISSUE THREAD

**Issue by mmarchegiani** [PocketCoffea/PocketCoffea#128]
From January 30th, the processors applying the muon scale factors are reporting the following error when trying to access the json correction file from `/cvmfs`:
```
FileNotFoundError: [Errno 2] No such file or directory: '/cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/MUO/2018_UL/muon_Z.json.gz'
```

By listing the corresponding folder, we find the following files:
```
(pocket-coffea) mmarcheg@t3ui02:/cvmfs/cms.cern.ch/rsync/cms-nanoAOD/jsonpog-integration/POG/MUO/2018_UL $ ll
total 147
-rw-r--r-- 1 cvmfs cvmfs  30683 Jan 30 05:05 muon_HighPt.json.gz
-rw-r--r-- 1 cvmfs cvmfs 119369 Jan 30 05:05 muon_Z_v2.json.gz
```
suggesting that the central MUO-POG files have been updated on 30.01.2024 at 5:05 am.
The straightforward fix is to change the path of the muon SF file in the defaults of pocket_coffea, but in the future a feature raising a warning in case of updated files from the json POG files could be implemented, such that the user is aware of changes in the correction files.

**Comment by andreypz** [comment]
This sounds like a bad practice from POG or jsonpog-intergation team. Files on CVMFS should never be changed. New version of the files should be uploaded when something is updated.

**Comment by mmarchegiani** [comment]
Exactly, and this is the case also for the files of other POGs, that have been updated on 30.01.2024 at 5:05 am.
We noticed it just because the files from the MUO-POG have changed path. I think it's something we should keep an eye on since the user could run the same analysis with different scale factors without noticing, just because a new version of corrections have been uploaded on `/cvmfs`.

**Comment by mmarchegiani** [comment]
The default muon SF for Ultra Legacy datasets have been updated: https://github.com/PocketCoffea/PocketCoffea/commit/8220cfbfd960140e06d2a86d28206a207e0a9645
